docs/documentation.html

```diff
@@ -60,6 +60,7 @@ <h3>Kafka 3.6 Documentation</h3>
                     <a href="/33/documentation.html">3.3.X</a>.
                     <a href="/34/documentation.html">3.4.X</a>.
                     <a href="/35/documentation.html">3.5.X</a>.
+                    <a href="/36/documentation.html">3.6.X</a>.
```

Comment:
one thing I don't completely understand is what/when builds this `/36/documentation.html` link in the website. I was following previous PRs' examples of how this version is bumped

Reply:
If I understand your question correctly, this is when the `/36/documentation.html` page was created: https://github.com/apache/kafka-site/pull/541
For `3.7` you'll need to create a PR like this as well, which will in turn create the `37` folder.

Reply:
ah, got it. That's done when we're closer to the end of the release

Reply:
https://github.com/apache/kafka-site/pull/576